theia-full-docker/next.package.json

```diff
@@ -124,6 +129,7 @@
         "vscode-builtin-icon-theme-seti": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/vscode-theme-seti-1.39.1-prel.vsix",
         "vscode-builtin-xml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/xml-1.39.1-prel.vsix",
         "vscode-builtin-yaml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/yaml-1.39.1-prel.vsix",
+        "vscode-clangd": "https://open-vsx.org/api/llvm-vs-code-extensions/vscode-clangd/0.0.21/file/llvm-vs-code-extensions.vscode-clangd-0.0.21.vsix",
```

Comment:
Good idea to add `vscode-clangd`. You should remove `@theia/cpp` from the `dependencies` section, since they both do similar. 

Can you do the same change in the `latest.package.json` and the `theia-cpp` image?

Reply:
I'm not so familiar with the different plugins/extensions from different sources... For me it was hard to judge what is really needed and what not... What plugin is responsible for the colorization of the source code? I thought, this is not the clangd extension... I can give it a try, if I see no problems I can remove ```@theia/cpp```.

Reply:
@marcdumais-work I have now tested the image without `@theia/cpp` and everything works as expected, even clanmg-tidy. So I updated the the clangd extension to the latest available version in the repository and made the chances as you proposed.

Reply:
Good thinking - indeed it used to be that `@theia/cpp` was providing the Textmate grammar responsible for syntax highlighting for `C/C++`. `vscode-clangd` does not provide it, since it's assumed to be present by default in VS Code. 

The grammar  is now contributed by this "built-in" VS Code extension, that we added a little while ago: https://github.com/theia-ide/theia-apps/pull/372/files#diff-fec21bf02d71557187accd444db76d3cL67